lib/Sabberworm/CSS/Parsing/SourceException.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace Sabberworm\CSS\Parsing;
+
+class SourceException extends \Exception {
+    private $iLineNo;
+    public function __construct($sMessage, $iLineNo = 0) {
+        $this->$iLineNo = $iLineNo;
```

Comment:
@sabberworm This was a fantastic catch! Sorry I didn't notice the extraneous `$`